### DISCUSSION TITLE
[Feature Request] Build number on main screen

### DISCUSSION THREAD

**Discussion by marcemira** [skyline-emu/skyline#1813] [Ideas]
**Scope:**
- 🚀 New Feature

# Summary
- Add support for displaying the build number on main screen, top header.

# Screenshots
![telegram-cloud-photo-size-1-4911382325628086908-y](https://user-images.githubusercontent.com/1788212/186235043-959a62b6-3c89-44e8-a0be-018e18060158.jpg)

**Comment by dima-xd** [comment]
I don't think it will be implemented anyhow because build version is from github workflow. Maybe it will be implemented only when ftx1 will be merged to master and builds will be from master branch

**Reply by marcemira** [reply]
Actually, I believe you can pass the `GITHUB_RUN_NUMBER` env variable to gradle at compile time, then I guess it's a matter of using it to display this meaningful number we all reference :)

**Reply by dima-xd** [reply]
> Actually, I believe you can pass the `GITHUB_RUN_NUMBER` env variable to gradle at compile time, then I guess it's a matter of using it to display this meaningful number we all reference :)

You can. But I don't think devs want to do this. Only real app version can be there. Now it's 0.0.3 but devs will reformat version numbering and then maybe they will add something like this

**Reply by marcemira** [reply]
Got it, and agreed, it would be way nicer to have a good SemVer goin on, with maybe GH Actions bumping accordingly the minors / patches, at least?

**Reply by dima-xd** [reply]
> 

I agree, it can be useful. It's easy to implement but we need to know what devs think about it

**Reply by nickbeth** [reply]
> Got it, and agreed, it would be way nicer to have a good SemVer goin on, with maybe GH Actions bumping accordingly the minors / patches, at least?

Yep, that's what we plan to do in the future.

**Comment by nickbeth** [comment]
Once we'll have proper versioning of release builds, we'll definitely add this. Thanks for the suggestion.
